addons/account/models/account_move.py

```diff
@@ -941,7 +941,7 @@ def _apply_cash_rounding(self, diff_balance, diff_amount_currency, cash_rounding
         others_lines -= existing_cash_rounding_line
         total_amount_currency = sum(others_lines.mapped('amount_currency'))
 
-        diff_balance, diff_amount_currency = _compute_cash_rounding(self, total_amount_currency)
+        diff_balance, diff_amount_currency = _compute_cash_rounding(self, self.currency_id.round(total_amount_currency))
```

Comment:
can you round the `amount_currency` in `_recompute_tax_lines` when creating the tax lines instead plz?